replicate-to-sql-with-generic-events/scripts/delete-data.js

```diff
@@ -0,0 +1,47 @@
+const { Kuzzle, WebSocket } = require('kuzzle-sdk');
+
+const kuzzle = new Kuzzle(new WebSocket('localhost'));
+
+const indexName = 'nyc-open-data';
+const collectionName = 'yellow-taxi';
+
+async function searchData() {
+  try {
+    const result = await kuzzle.document.search(
+      indexName,
+      collectionName,
+      {},
+      { size: 500 }
+    );
+    return result.hits;
+  } catch (error) {
+    return [];
+  }
+}
+
+async function deleteData(ids = []) {
+  try {
+    const result = kuzzle.document.mDelete(indexName, collectionName, ids);
+    await kuzzle.collection.refresh(indexName, collectionName);
```

Comment:
You don't need to refresh the collection here since you didn't make any search after the delete